src/app/content/studyGuides/components/StudyGuides.tsx

```diff
@@ -0,0 +1,62 @@
+import { HTMLElement } from '@openstax/types/lib.dom';
+import React from 'react';
+import { useSelector } from 'react-redux';
+import styled from 'styled-components';
+import { typesetMath } from '../../../../helpers/mathjax';
+import Loader from '../../../components/Loader';
+import { useServices } from '../../../context/Services';
+import theme from '../../../theme';
+import { assertWindow } from '../../../utils';
+import allImagesLoaded from '../../components/utils/allImagesLoaded';
+// Temporary import from /highlights directory until we make all this logic reusable and move it to content/
+import * as Styled from '../../highlights/components/ShowMyHighlightsStyles';
+import { SectionHighlights } from '../../highlights/components/SummaryPopup/SectionHighlights';
```

Comment:
I think the best option would be to move `SummaryPopup` into `content/components` because most of these components like "filters", "chapter selectors", "go to top button", etc are reusable between "my highlights" and "study guides", but I wanted to wait for a review to make sure you don't have anything against it.

Reply:
we've been moving some popup stuff to app/components, you can put anything not content specific there, like the back to top button, or generic filter stuff.

the "Chapter filter" specifically, is the only piece that is content specific, so that should not go in app/components, but content/components would be fine

Reply:
I've ended up with moving, `SectionHighlights`, `GoToTopButton`, some `constants`, and move styles from `ShowMyHighlightsStyles` to associated components